### DISCUSSION TITLE
Welcome to Driwwwle Discussions!

### DISCUSSION THREAD

**Discussion by itsnitinr** [itsnitinr/driwwwle#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Duckie08** [comment]
Hi, i am trying to clone this project and got some troubles, please support me :3

**Reply by itsnitinr** [reply]
Hey there! How can I help you? Can you share what trouble or error you're facing?

**Comment by brandmetoday** [comment]
Hi, I am a designer (little dev knowledge) and testing your code. I am facing a problem in the username section while signup. It showing "This username is invalid or not available"

**Reply by itsnitinr** [reply]
Hi @brandmetoday, could you please let me know what username are you entering?

**Reply by brandmetoday** [reply]
Got it cleared. thanks for your quick reply.
